src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages_fi.properties

```diff
@@ -9,7 +9,7 @@ javadoc.empty=Javadoc on tyhjä §.
 javadoc.expectedTag=Javadoc-tagi puuttuu: {0} tagi ''{1}''.
 javadoc.extraHtml=Extra HTML-koodi löytyy: {0}
 javadoc.incompleteTag=Puutteelliset HTML-koodi löytyy: {0}
-javadoc.invalidInheritDoc=Virheellinen käyttö '{'inheritDoc '}' tag.
+javadoc.invalidInheritDoc=Virheellinen käyttö ''inheritDoc'' tag.
```

Comment:
If we are going to sync all these tags, then I think they should be consistent. This just has the tag name while others have the `@`.

Reply:
Fixed it, also cross-checked the changes with https://github.com/checkstyle/checkstyle/commit/50675ca340f9d7cc1a55bd2d12108a8b6c35e078